invenio_app/config.py

```diff
@@ -34,6 +34,13 @@
    configuration.
 """
 
+RATELIMIT_KEY_FUNC = None
+"""Define custom key function.
+
+    This config isnot part of Flask limiter.
```

Comment:
Typo `is not`. Could you be just more explicit? Just reading this, what is a `key func`? :)